src/tomato_bronkhorst/__init__.py

```diff
@@ -0,0 +1,405 @@
+from typing import Any
+from propar import PP_TYPE_FLOAT
+from propar import instrument as Instrument
+from tomato.driverinterface_1_0 import Attr, ModelInterface, Task
+
+
+# you mentionned that the property map should be outside, check
+# but also that the parameters were hard coded, for more I think you asked to handle this :
+# here is a proposition for more flexibility :
+class PropertyMap:
```

Comment:
I don't see the benefit of having this as a `Class`, since there are no useful class methods in it. I'm not sure we'll ever need to add a property from within tomato...